.gitignore

```diff
@@ -25,7 +25,7 @@ cachegrind.out*
 # Codelite (among others) project files
 *.project
 # ctags index files
-tags
+*tag*
```

Comment:
Ah, sounds very reasonable then. May I suggest to open a separate PR for this? We try to keep every PR focused on one logical feature to simplify the reviewing...